configure

```diff
@@ -1152,15 +1168,15 @@ function PARTITIONING_DEVICES_label() {
 
 	if [[ "$invalid" -gt 0 ]]; then
 		echo " └ (${#PARTITIONING_DEVICES[@]} devices, \Z1$invalid invalid\Zn)"
-	elif [[ "${#PARTITIONING_DEVICES[@]}" -eq 1 && "$PARTITIONING_SCHEME" == "raid0_luks" ]]; then
+	elif [[ "${#PARTITIONING_DEVICES[@]}" -eq 1 && "$PARTITIONING_SCHEME" == "raid0_luks" || "$PARTITIONING_SCHEME" == "raid1_luks" ]]; then
```

Comment:
`&&` binds stronger than `||`, so you probably want to add parenthesis around the second part, otherwise the `-eq 1` is only carried out for raid0